### ISSUE TITLE
timeout issue

### ISSUE THREAD

**Issue by caseyjp11** [M-Reimer/wine-lol#101]
The problem happens once or twice a session (5 or 6 matches) AFTER finishing a fight. The session change from full screen back to the results-table windowed screen doesn't happen. The command line reports this:

'0178:err:ntdll:RtlpWaitForCriticalSection section 0x546d0bc "?" wait timed out in thread 0178, blocked by 0190, retrying (60 sec)'

and it repeats until killed. A restart shows that events are recorded, so this is purely a client-side issue.
AMD FX6300 CPU
AMD RX550 (Lexa Pro) GPU
16 gig ram
gigabyte motherboard.
Archlinux latest production updates
KDE -latest.

**Comment by M-Reimer** [comment]
All I can do is keep this open here so it is documented and (probably) acts as a discussion thread if someone else finds this Issue. It will, for sure, not be fixed by me.

Maybe report this Issue here too: https://www.reddit.com/r/leagueoflinux/

**Comment by M-Reimer** [comment]
Replacing "wontfix" with "helpwanted" because maybe someone else wants to use this Issue as discussion platform for fixing this problem. All I want to express is that I won't do anything to fix such issues.

**Comment by caseyjp11** [comment]
> All I can do is keep this open here so it is documented and (probably) acts as a discussion thread if someone else finds this Issue. It will, for sure, not be fixed by me.
> 
> Maybe report this Issue here too: https://www.reddit.com/r/leagueoflinux/

I'll put that up a bit later today. I thought you should know about it. It isn't a show stopper as it only happens during that particular session change, but something is definitely buggin' up. :-)

**Comment by DanRiffo** [comment]
Hi, I use a similar setup:

Arch linux fully updated + KDE
FX-6300 @4GHz 
12Gb RAM
970 chipset Gigabyte Motherboard
Radeon HD7770 1GB GDDR5 (old, I know)
AMDGPU + AMDVLK

I experienced some problems on fullscreen, so I use the game as Borderless instead. Can alt+tab freely without issues.
Maybe you set the League launcher to close automatically while in game and it fails to restart sometimes after a game.
I use my launcher on "low spec mode" and "never close client during game" (see attachment)

![lol_launcher](https://user-images.githubusercontent.com/37986576/154851700-c803c803-c6b7-4756-88d5-6cbf73a73d87.png)


Hope it helps :D

**Comment by M-Reimer** [comment]
Mass closing issues. My PKGBUILD builds a much more recent wine version now so some issues may be obsolete now.
